src/blockchains/evm/contracts/SovrynHelper.ts

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
```

Comment:
I think everything related to dex should be under dex/ directory, you can create `dexes/soveryn/contracts/` if you want to have detailed separation.

Reply:
Sounds good, I think that everything in `dexes/sovryn` makes it simpler and more readable.